locales/wxMaxima/fr.po

```diff
@@ -2637,7 +2637,7 @@ msgstr ""
 #: ../../src/ConfigDialogue.cpp:179
 #, fuzzy
 msgid "Finnish"
-msgstr "Danois"
+msgstr "Finlandais"
```

Comment:
This translation will not be used as it is fuzzied. Can you use specialized software (poEdit, Lokalize, Virtaal) for translations? Thanks in advance for your work.